crates/network-scanner-net/examples/drop_runtime.rs

```diff
@@ -0,0 +1,92 @@
+use std::{net::SocketAddr, sync::Arc};
+
+#[tokio::main]
+pub async fn main() -> anyhow::Result<()> {
+    utils::start_server();
+    tracing_subscriber::fmt::SubscriberBuilder::default()
+        .with_max_level(tracing::Level::TRACE)
+        .with_thread_names(true)
+        .init();
+    {
+        let async_runtime = network_scanner_net::runtime::Socket2Runtime::new(None)?;
+        {
+            let good_socket = async_runtime.new_socket(socket2::Domain::IPV4, socket2::Type::STREAM, None)?;
+            let bad_socket = async_runtime.new_socket(socket2::Domain::IPV4, socket2::Type::STREAM, None)?;
+
+            tracing::info!("good_socket: {:?}", good_socket);
+            tracing::info!("bad_socket: {:?}", bad_socket);
+
+            let available_port = 8080;
+            let non_available_port = 12345;
```

Comment:
nitpick: Strictly speaking, if the server is listening on the port 8080, the port is not "available" because it’s "in use", so this naming may be misguiding. `echo_server_port` and `unused_port` may be better alternatives.

Reply:
updated